### PR TITLE
Added the aws credentials to github

### PR DESCRIPTION
**Pull request by nandeeshsb9**

### CODE REVIEW

**Review by vietnguyengit — CHANGES_REQUESTED** [review]
same here, please tidy up the commits history, thank you

File: ogcapi-java-dev-td.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{
```
Comment (utas-raymondng) [inline ogcapi-java-dev-td.json:1]: Is there way to integrate docker-compose.yml together, cost dev need to maintain two very similar config one for docker one for aws tasks

**Review by utas-raymondng — CHANGES_REQUESTED** [review]

File: .github/workflows/github-actions.yml [hunk 2]
```diff
@@ -0,0 +1,186 @@
+# This is a basic workflow to help you get started with Actions
+name: Connect to an AWS role from a GitHub repository
+
+# Controls when the action will run. Invokes the workflow on push events but only for the main branch
+on:
+  push:
+    branches: [5165-cicd]
```
[the remark this answers is not in the document]
Comment (priyaranjanpatil) [reply .github/workflows/github-actions.yml:7]: After testing the deployment will have to point to main branch only for PR and push

**Comment by priyaranjanpatil** [comment]
Closing this PR and will create a new one after rebase and squash with few env var config changes

**Pull request closed without merging**